lib/index.js

```diff
@@ -169,10 +169,18 @@ Assertion.prototype.maps = function(fixture, settings, options){
   actual = JSON.parse(JSON.stringify(actual));
 
   if (options.ignored) {
-    options.ignored.map(function(name) {
-      delete actual[name]
-      assert(!expected[name], "ignored expected value:" + name);
-    });
+    // Recursively strip all keys given in options.ignored from actual, and from
+    // all objects contained in or nested under actual.
+    var stripIgnoredFromResult = function(obj) {
+      for(prop in obj) {
```

Comment:
Nitpicky, but `for`, `if` should probably have a space between the parenthesis. I don't think we're linting in these repos